client/views/list/list.html

```diff
@@ -10,7 +10,14 @@
 		</div>
```

Comment:
Let's fix the code formatting here as well.

Reply:
Updated the formatting.